sphinx/writers/latex.py

```diff
@@ -295,7 +296,9 @@ class LaTeXTranslator(nodes.NodeVisitor):
         'classoptions':    '',
         'extraclassoptions': '',
         'passoptionstopackages': '',
-        'inputenc':        '\\usepackage[utf8]{inputenc}',
+        'inputenc':        ('\\ifPDFTeX\n'
+                            '  \\usepackage[utf8]{inputenc}\n'
+                            '\\else\\fi'),
```

Comment:
you could use `'\\fi'),` here, the `\\else` is not needed but I understand you use it before next lines do.